.github/workflows/test-on-projects.yml

```diff
@@ -4,23 +4,18 @@ on:
   push:
     branches:
       - main
+      - github-testing
```

Comment:
Could you please delete this branch from yml file?
Probably we should run this action on push and pr into main branches